### DISCUSSION TITLE
Does celery support class method task binding?

### DISCUSSION THREAD

**Discussion by realpipi** [celery/celery#7085] [Q&A]
In OOD, we usually have multiple class methods inside a class, and those methods could be only called from the class object. Sometimes polymorphism will be applied in the design too.

I am wondering if celery can wrap class methods as tasks and bind different tasks under a class to different queues? If not, how can I achieve this?

Thanks in advance!

[a comment by an automated account is not shown]

**Comment by thedrow** [comment]
Class methods or instance methods?
